examples/remote-settings-cli/Cargo.toml

```diff
@@ -12,3 +12,10 @@ log = "0.4"
 clap = {version = "4.2", features = ["derive"]}
 anyhow = "1.0"
 env_logger = { version = "0.10", default-features = false, features = ["humantime"] }
+reqwest = { version = "0.12", features = ["json"] }
+serde_json = "1"
+futures = "0.3"
+indicatif = "0.17"
+tokio = { version = "1", features = ["full"] }
+serde = { version = "1.0.205", features = ["derive"] }
+chrono = "0.4.26"
```

Comment:
I'm slightly worried about moz-central vendoring with all these new packages.  Can you test vendoring and see if there are version mismatches and/or missing audits.